projects/github-actions/test-results-to-slack/action.yml

```
@@ -0,0 +1,29 @@
+name: "Send tests results to Slack"
+description: "Send Slack notifications with test results."
+branding:
+  icon: 'activity'
```

Maybe we could use `message-square` here?

```suggestion
  icon: 'message-square'
```